node/test/mocktests.ts

```diff
@@ -183,5 +184,100 @@ describe('Mock Tests', function () {
         
         assert(tool, "tool should not be null");
         assert(rc == 0, "rc is 0");
-    })                
+    })
+    
+    it('Mock toolRunner returns correct output', async () => {
+        const expectedStdout = "atool output here" + os.EOL + "abc" + os.EOL + "junk";
```

Comment:
Wouldn't/shouldn't the real task runner emit 'stdline' for "junk"?

Reply:
https://github.com/Microsoft/azure-pipelines-task-lib/blob/2f0dde7e1786bcca01f0e8a0d6bdaa760b89ea8f/node/toolrunner.ts#L667

Reply:
Yes, and it does [on close](https://github.com/Microsoft/azure-pipelines-task-lib/blob/2f0dde7e1786bcca01f0e8a0d6bdaa760b89ea8f/node/toolrunner.ts#L666). Sorry, looks like my brain didn't quite get going this morning :)

Reply:
Should actually be fixed now